lib/pbench/test/unit/server/test_upload.py

```diff
@@ -227,16 +227,16 @@ def test_bad_metadata_upload(self, client, server_config, pbench_drb_token):
                     "Content-MD5": "ANYMD5",
                 },
                 query_string={
-                    "metadata": "global.xyz#A@b=z:y,foobar.badpath:data,server.deletion:3000-12-25T23:59:59+00:00"
+                    "metadata": "global.xyz#A@b=z:y,foobar.badpath:data,server.deletion:'3000-12-25T23:59:59+00:00'"
                 },
```

Comment:
So, this is a breaking change, eh?  (I.e., any values which have colons in them must now be quoted?)

Reply:
Colon or comma, yeah. 😀 That's a big reason I'd really like to get this in before we declare "1.0" because it wouldn't feel good otherwise.